# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-python-extras/gdl-api.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gnome-python-extras-2.14.3-orig/gdl/gdl.override gnome-python-extras-2.14.3/gdl/gdl.override
--- gnome-python-extras-2.14.3-orig/gdl/gdl.override	2009-01-21 07:28:44.000000000 +0000
+++ gnome-python-extras-2.14.3/gdl/gdl.override	2009-01-21 07:31:22.000000000 +0000
@@ -6,7 +6,7 @@
 #include <pygtk/pygtk.h>
 #include <gdl/gdl-dock.h>
 #include <gdl/libgdltypebuiltins.h>
-#include <gdl/gdl-icons.h>
+#include <gdl/gdl-stock-icons.h>
 #ifndef HAVE_GDL_0_7
 # include <gdl/gdl-dock-bar.h>
 #endif
